src/rpc/blockchain.cpp

```diff
@@ -793,7 +793,8 @@ static RPCHelpMan getblockfrompeer()
         "\nAttempt to fetch block from a given peer.\n"
         "\nWe must have the header for this block, e.g. using submitheader.\n"
         "Subsequent calls for the same block and a new peer will cause the response from the previous peer to be ignored.\n"
-        "\nReturns an empty JSON object if the request was successfully scheduled.",
+        "\nReturns an empty JSON object if the request was successfully scheduled."
+        "\nNote: On a pruned node this block might be pruned again as soon as the pruneheight surpasses the blockheight at the time of fetching the block.\n",
```

Comment:
not sure, but without the spaces these look like RPC fields or inputs, yet aren't part of this RPC 
```suggestion
        "\nNote: On a pruned node this block might be pruned again as soon as the prune height surpasses the block height at the time of fetching the block.\n",
```

Reply:
done